documentation/sphinx/source/release-notes/release-notes-620.rst

```diff
@@ -4,6 +4,11 @@
 Release Notes
 #############
 
+6.2.28
+======
+* Log detailed team collection information on trigger. `(PR #3912) <https://github.com/apple/foundationdb/pull/3912>`_
```

Comment:
sorry I didn't spot this out. 
Can we the trigger condition more explicit in the note? something like the following:
`Log detailed team collection information when storage servers' medianAvailableSpace is less than TARGET_AVAILABLE_SPACE_RATIO)`

Reply:
We don't usually write release notes with so much internal detail. May something like `... when the median available space on a storage server gets low`.

Reply:
I don't think this is accurate, as what I added literally is a way to trigger the logging that can be used in other cases too.

Reply:
I don't know if there are other ways this can trigger now, but another typical property of release notes is to describe things as they would be perceived by a user or operator of the cluster. If the facility is currently only being triggered in the case described above, then that probably makes a reasonably good release note even if we also provided additional capabilities for developers.

Reply:
I agree with AJ. 
Consider this scenario:
SRE reads the release note and found this new feature. Now he/she asks what is the condition to trigger? 
If we don't describe it in release note, they will have to search the source code, which I don't think is friendly.

Reply:
I was thinking of adding a CLI command to allow user to manually trigger the logging for a better understanding of the state of data distribution. However I now think that might be an overkill for now. Will change release notes.